### PR TITLE
Adjust tooltips and descriptions in the campaign missions

### LINKED ISSUE #15662: Tech structures show their tooltips on campaign missions where they aren't capturable
E.g. on allies03a

screenshot 2018-09-30 at 14 07 40

### PR DESCRIPTION
**Pull request by abcdefg30**
Closes #15662.

### CODE REVIEW

File: mods/ra/rules/campaign-rules.yaml [hunk 1]
```diff
@@ -44,3 +44,13 @@ E7.noautotarget:
 
 ^Vehicle:
 	Demolishable:
+
+FCOM:
+	ExternalCapturable:
```
Comment (pchote) [inline mods/ra/rules/campaign-rules.yaml]: This will conflict with #15661

**Review by pchote — COMMENTED** [review]
LGTM otherwise

**Comment by MustaphaTR** [comment]
Since this is technically a regression/mistake from addition of Tooltip Description, which is new in the release, imo it would be nice to include this in the HotFix, at least the first commit.

File: mods/ra/rules/campaign-rules.yaml (hunk 1 above)
Comment (abcdefg30) [reply mods/ra/rules/campaign-rules.yaml]: I'll wait for that PR and then rebase this one.

**Comment by pchote** [comment]
Ping @abcdefg30.

**Comment by pchote** [comment]
Ping @abcdefg30.

**Comment by abcdefg30** [comment]
Rebased & updated.

**Review by reaperrr — DISMISSED** [review]

File: mods/ra/rules/campaign-rules.yaml [hunk 2]
```diff
@@ -52,3 +52,14 @@ MONEYCRATE:
 		Amount: 2000
 	RenderSprites:
 		Image: scrate
+
+FCOM:
+	Capturable:
+		Types: ~disabled
+	-GivesBuildableArea:
+	-BaseProvider:
```
Comment (pchote) [inline mods/ra/rules/campaign-rules.yaml]: This removal was missing, so I pushed over the branch with the fix.

**Review by pchote — APPROVED** [review]

**Comment by pchote** [comment]
Picked to prep: a8c57beebfc2c9528af5c8814b04c32de5c50876, 3fed03df845c28f1acb076d4ad27ae325c4f39a7, 105fd83520d70cde73176fc529d2dda8159b7939

https://github.com/OpenRA/OpenRA/wiki/Changelog/_compare/8b0a6cc71d84998635a64aebe53925e1d9fe5485

**Pull request merged**